### PR TITLE
Allow sylius 1.10 | update GH Actions

### PR DESCRIPTION
**Pull request by TonySma**
Allow Sylius ^1.10.0-beta1
Update GH Matrix

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -16,16 +16,27 @@ jobs:
...
-          sudo update-alternatives --set php /usr/bin/php${{ matrix.php }}
-          php -v
+      -
+        name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+            php-version: "${{ matrix.php }}"
+            extensions: intl
+            tools: symfony
+            coverage: none
```
Comment (jacquesbh) [inline .github/workflows/tests.yml:39]: Any reason why you are changing this?

PHP 8.0 is installed on the runner. See https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-software.

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -44,8 +55,13 @@ jobs:
           curl https://get.symfony.com/cli/installer | bash
           echo ${HOME}/.symfony/bin >> $GITHUB_PATH
 
+      -
+        name: Restrict Sylius version
+        if: matrix.sylius != ''
+        run: composer require "sylius/sylius:${{ matrix.sylius }}" --no-update --no-scripts --no-interaction
+
```
Comment (jacquesbh) [inline .github/workflows/tests.yml:62]: Nice!

```suggestion
      - name: Restrict Sylius version
        run: composer require "sylius/sylius:${{ matrix.sylius }}" --no-update --no-scripts --no-interaction
```

**Review by jacquesbh — CHANGES_REQUESTED** [review]
Thanks @TonySma for your PR!

Some things I would like to understand and maybe some code changes.

File: .github/workflows/tests.yml (hunk 1 above)
Comment (TonySma) [reply .github/workflows/tests.yml:39]: https://github.com/TonySma/SyliusSettingsPlugin/runs/2906412101?check_suite_focus=true
Strangely GH seems to remove ".0" of 8.0 and it fail when alternative update. 

I just pick the Sylius method instead: 
https://github.com/Sylius/Sylius/blob/master/.github/workflows/packages.yml#L61

**Comment by TonySma** [comment]
Just updated composer.json to v1.10.0 thanks to the sylius release

File: .github/workflows/tests.yml (hunk 1 above)
Comment (TonySma) [reply .github/workflows/tests.yml]: Fixed with quotes

**Comment by jacquesbh** [comment]
Also @TonySma, what about the second part of the tests to allow PHP 8 and Sylius 1.10 as well?

**Comment by TonySma** [comment]
Seem's ok to me now 
https://github.com/TonySma/SyliusSettingsPlugin/runs/2952100589?check_suite_focus=true

**Review by jacquesbh — APPROVED** [review]

**Comment by jacquesbh** [comment]
Thank you @TonySma!! 🥰

**Comment by vvasiloi** [comment]
@jacquesbh When do you plan to release this?

**Pull request merged**